### PR TITLE
Golang package renaming

### PR DESCRIPTION
**Pull request by xanimo**
Fixes # .

Replaced coinbase/rosetta-bitcoin with rosetta-dogecoin/rosetta-dogecoin in both go.mod and Makefile.

### CODE REVIEW

**Comment by xanimo** [comment]
Apparently my markup skills are rusty as my question didn't get posted.  I mentioned I am still ironing out kinks in the commit, submit PR process as you can see my 'reverts'.  I'm getting it I think.  I was trying to omit the Dockerfile I was working on and will manually add and commit files via cmd instead of VS Code where I have to push everything I've worked on.  Regardless any feedback appreciated, hopefully this is what you were looking for but let me know if not.

**Comment by IncognitoJam** [comment]
You might want to squash some of those commits with rebase 😅

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/coinbase/rosetta-bitcoin
```
Comment (IncognitoJam) [inline go.mod:1]: I don't think renaming the package is as easy as changing this, as the old package name is referened in lots of places (see [main.go](https://github.com/rosetta-dogecoin/rosetta-dogecoin/blob/main/main.go#L31))

Comment (xanimo) [reply go.mod:1]: > You might want to squash some of those commits with rebase 😅

Good call, sorry I'm rusty.  Should I add cancel this rebase and resubmit?

Comment (IncognitoJam) [reply go.mod:1]: No it's okay! After you've rebased locally just force-push to this branch and the PR will still be here

**Comment by patricklodder** [comment]
That last commit 9557152 seems to have created a bit of a chaos with what looks like line endings.

File: go.mod (hunk 1 above)
Comment (xanimo) [reply go.mod:1]: > No it's okay! After you've rebased locally just force-push to this branch and the PR will still be here

Ok learning moment, I git rebase -i 421cd95bdfea2079a412e4f3f06855e0d43ad600 to my last SHA and am wondering why it adds changes to all files +11,777 -11,777.  I understand it flattens the repo to reflect one commit but did I do it right?

**Comment by xanimo** [comment]
> That last commit [9557152](https://github.com/rosetta-dogecoin/rosetta-dogecoin/commit/9557152f9c63a3277f6088051e898c5b9c3be2f8) seems to have created a bit of a chaos with what looks like line endings.

I'm going to start over from scratch lol, apologies.  I'm doing development on Windows with WSL2 and Ubuntu.

**Pull request closed without merging**